variables.tf

```diff
@@ -44,3 +49,9 @@ variable "security_group_ids" {
   description = "The security group ids which can be given to the bastion instance, defaults to empty"
   default     = []
 }
+
+variable "extra_tags" {
```

Comment:
also please rename this parameter just to `tags`.

Reply:
In this case we can use it together with https://github.com/Flaconi/terraform-null-tags module

Reply:
Makes sense.